Gruntfile.js

```diff
@@ -38,12 +38,26 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['clean:dist', 'sass', 'webpack']);
 
+  // Regular grunt server, used when running inside the payments-env docker.
   grunt.registerTask('serve',
-    ['clean:dist', 'sass:dev', 'webpack-dev-server:start', 'watch:sass']);
-  grunt.registerTask('start',
     ['clean:dist', 'sass:dev', 'webpack:dev', 'watch:sass']);
-  grunt.registerTask('start-email',
+  // The dev server used for live reloading in the browser.
+  grunt.registerTask('serve:dev',
```

Comment:
Using `serve` for the standard case and email is a bit odd because nothing is served by grunt wheres in the webpack dev-server case it is actually serving something on a port.

I think `start` vs `serve` are already clearly delineated - maybe the issue here is more docs. 

`watch` might make more sense than `start` but watch is already a task so you can't use `grunt watch` / `grunt watch-email`. We could do `watch-static` and `watch-email` though.

Maybe we should do better docs in a table on the README so it's super clear?

E.g:

## Grunt tasks

Each task is run with the grunt command e.g. `grunt watch-static`.

| Task name       | Description |
| ------------ | --------------- | 
| serve  | Watching the CSS/JS for changes and serve the static files via webpack dev server to enable hot module reloading of JS |
| watch-static | watch the files for changes and update CSS/JS on changes |
| watch-email | watch the email specific static files for changes and update CSS/JS on changes |